src/Hybridauth.php

```diff
@@ -134,7 +134,7 @@ public function getAdapter($name)
                 }
             }
             if ($adapter === null) {
-                throw new InvalidArgumentException('Unknown Provider.');
+                throw new InvalidArgumentException("Unknown Provider ($adapter).");
```

Comment:
not sure, see if above, it checks if adapter is null, so maybe here you should log $name